BUILD.bazel

```diff
@@ -10,7 +10,17 @@
 # limitations under the License.
 
 load("@com_github_bazelbuild_buildtools//buildifier:def.bzl", "buildifier")
+load("@bazel_gazelle//:def.bzl", "gazelle")
 
 buildifier(
     name = "buildifier",
 )
+
+# gazelle:prefix github.com/magma/magma
```

Comment:
In the PR description, can you enumerate the precise shell commands you ran to construct this PR (to the extent you used tooling)? So others can repeat it if necessary. E.g. I think you indicated that for now we track off of go.mod -> need to re-run some tooling to re-sync regularly?

Reply:
I see you have some information in the README.md in the experimental directory - if that's sufficient (going to review now) maybe just call it out in PR description?

Reply:
OK so re-reading your PR description in fact you do have this already there =D